Makefile

```diff
@@ -20,5 +20,7 @@ mod/replace/atmo:
 
 tidy:
 	go mod tidy && go mod download && go mod vendor
+lint:
```

Comment:
Let's add `lint` to the prerequisites of `subo` and all of the `subo/*` targets. I don't think it will get in the way of any CI steps.

Reply:
good idea, thank you! fixed in https://github.com/suborbital/subo/pull/189/commits/72e7d42b68f25969a4cc64c0f0650033bbf411fa